servers/src/main/java/tachyon/worker/block/BlockMetadataManagerView.java

```diff
@@ -161,7 +162,7 @@ public StorageTierView getTierView(int tierAlias) {
    * @return the list of StorageTierViews
    */
   public List<StorageTierView> getTierViews() {
-    return mTierViews;
+    return Collections.unmodifiableList(mTierViews);
```

Comment:
The caller of this method cannot modify the list, i.e. add/remove items. I think it's safer because this class only provides a 'view'.